group_vars/all

```diff
@@ -28,7 +28,11 @@ galaxy_extras_apt_package_state: present
 tool_dependency_dir: /home/{{ galaxy_user_name }}/tool_dependencies
 shed_tools_dir: "{{ galaxy_server_dir }}/../shed_tools"
 tool_data_dir: "{{ galaxy_server_dir  }}/tool-data"
-additional_files_list: []
+
+additional_files_list:
+    - { src: "extra-files/tool_sheds_conf.xml", dest: "{{ galaxy_config_dir }}" }
+    - { src: "extra-files/dependency_resolvers_conf.xml", dest: "{{ galaxy_config_dir }}/dependency_resolvers_conf.xml" }
```

Comment:
See also here: https://github.com/galaxyproject/galaxy/pull/2801#issuecomment-240203913. It does log a small error when starting up, but use the default resolver order.